entry.go

```diff
@@ -179,16 +180,25 @@ func (te *TunaEntry) StartReverse(stream *smux.Stream) error {
 		return err
 	}
 
-	getTotalCost := func() common.Fixed64 {
+	getTotalCost := func() (common.Fixed64, common.Fixed64) {
+		in := common.Fixed64(atomic.LoadUint64(&te.reverseBytesIn))
+		out := common.Fixed64(atomic.LoadUint64(&te.reverseBytesOut))
+		time.Sleep(TrafficCalDelay * time.Second)
```

Comment:
We shouldn't add delay here. Instead, it should be added to caller of `getTotalCost`